RHDEVDOCS-5615: Updating the installing build strategies section (#64609)
openshift/openshift-docs: modules/ob-installing-buildah-strategy.adoc

[id="installing-buildah-strategy_{context}"]
= Installing buildah strategy

> You can install the `buildah` cluster build strategy, which does not require a daemon to run and is available for use by unprivileged users. By using the `buildah-strategy-managed-push` resource for installing `buildah`, you can push the image to the target repository without any further processing.

Reviewer: ```suggestion
You can install the `buildah` cluster build strategy, which does not require a daemon to run and is available for unprivileged users. When using the `buildah-strategy-managed-push` resource for installing `buildah`, you can push the image to the target repository without any further processing.
```
